src/chaplin/lib/utils.coffee

```diff
@@ -89,9 +88,20 @@ utils =
       (moduleName, handler) ->
         enqueue -> handler require moduleName
 
+  # DOM helpers
+  # -----------
+
+  matchesSelector: do ->
+    $ = document.body
```

Comment:
Changed that to `el = document.documentElement`. `body` may not be present if Chaplin.js is blocking script in `head`, or it is legacy frame document or SVG file.